apiserver/test-apiserver.sh

```diff
@@ -18,12 +19,42 @@ export PYTHONPATH=$(ls $SPARK_HOME/python/lib/py4j-*-src.zip):$SPARK_HOME/python
 export JAR=../hail/build/libs/hail-all-spark.jar
 export PYSPARK_SUBMIT_ARGS="--conf spark.driver.extraClassPath=$JAR --conf spark.executor.extraClassPath=$JAR pyspark-shell"
 
-python3 apiserver/apiserver.py &
+HAIL_JWT_SECRET_KEY_FILE=jwt-test-secret-key python3 apiserver/apiserver.py &
 server_pid=$!
 
 ../until-with-fuel 30 curl -fL http://localhost:5000/healthcheck
 
-export HAIL_TEST_SERVICE_BACKEND_URL=http://localhost:5000
+export HAIL_APISERVER_URL=http://localhost:5000
+export HAIL_TOKEN_FILE="$(pwd)/jwt-test-user-token"
 
 python3 -m unittest test.hail.table.test_table.Tests.test_range_table
 python3 -m unittest test.hail.linalg.test_linalg.Tests.test_matrix_ops
```

Comment:
These aren't running against the apiserver, that's what HAIL_TEST_SERVICE_BACKEND_URL is for.  See test/hail/helpers.py::startTestHailContext.

Reply:
Huh. But when you init a hail context it checks if HAIL_APISERVER_URL is set, if so it creates a service backend. I did not see the test code before. I agree the apiserver was being properly tested before. 

https://github.com/hail-is/hail/blob/master/hail/python/hail/context.py#L70

What’s the thinking on having another variable for the tests?

@akotlar discussion continuing here.

Reply:
It does seem reasonable to have a single url variable, set based on testing/deployment context. Wouldn’t this also allow us to simplify the helper, and just make one unit call?